post-scan-actions/azure-python-slack-notification/template.json

```diff
@@ -0,0 +1,233 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "fssSlackFunctionName": {
+            "type": "string",
+            "defaultValue": "[concat('fssslack', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "The name of the slack function to make."
+            }
+        },
+        "webHookURL": {
+            "type": "string",
+            "defaultValue": "[concat('fssslack', uniqueString(resourceGroup().id))]",
```

Comment:
It's not a reasonable default value for the URL. `defaultValue` can be removed to indicate it's a required parameter.
```suggestion
```

Reply:
removed defaultValues for both slackChannelName and slackWebHookURL